packages/integrations/components/pico/index.jsx

```diff
@@ -63,9 +63,10 @@ const Pico = (props) => {
 
   const irvingIsLoading = useLoading();
   const { ready: picoReady } = useSelector(picoLifecycleSelector);
-  const scriptLoaded = window?.Pico?.getInstance()?.scriptLoaded;
   const contentReady = useSelector(picoContentReadySelector);
   const visited = useSelector(picoVisitedSelector);
+  const picoInstance = window?.Pico?.instance || false;
+  const scriptLoaded = picoInstance ? window?.Pico?.getInstance() : false;
```

Comment:
Does this still need to check for `.scriptLoaded`? Like

```suggestion
  const scriptLoaded = picoInstance ? picoInstance.scriptLoaded : false;
```

Reply:
Yes. The instance in this case is sale, which was the root issue here.